SR2024 kick off email (#66)
srobo/volunteer-emails: SR2024/2023-07-15-sr2024-kickofff.md

Hi All,

> Planning for SR2024 is underway! We're still in the early stages so this is the perfect time for you to get more involved.

Reviewer: ```suggestion
Planning for SR2024 is underway! With Kickstart just around the corner now is a great time to get involved.
```

## Key Dates / Programme Structure

Our experiment with adding a virtual element to the physical competition last year was a success, so we're building on what we've learnt and will continue to have both a physical and virtual competition.

We've pencilled in some key dates for the year with some of the key ones listed below. Be sure to check out the SR shared calendar for all the planned dates including Tech Days and prep events.

- 2023-10-21 Kickstart
- 2024-02-10 Virtual competition (provisional)
- 2024-03-23 – 2024-03-24 Competition (provisional)

## Get involved

There's loads of ways to help with SR2024, taking as much or as little time as you can give.

> - The physical competition needs a small army of volunteers to run in numerous technical and non technical roles. We'll open sign ups for volunteering at the event as soon as we can confirm the venue ???

Reviewer: ```suggestion
- The physical competition needs a small army of volunteers to run in numerous technical and non technical roles. We'll open sign ups for volunteering at the event as soon as we can confirm the venue later in the year

```

- 2024-03-23 – 2024-03-24 Competition (provisional)

## Get involved

There's loads of ways to help with SR2024, taking as much or as little time as you can give.

- The physical competition needs a small army of volunteers to run in numerous techincal and non technical roles. We'll open sign ups for volunteering at the event as soon as we can confirm the venue ???
- We're on the look out for mentors, who would regularly attend a teams' meetings (either virtually on in person) and assist them
- We'll be running Tech Days and in person events which will need people to help run

Keep an eye on your inboxes for information about these opportunities as details are announced!

### Teams

Running Student Robotics is no small feat so to help spread the load we've organised into teams, each with their own area of interest.

Committee| Description | Email | Current Meeting Time (UK time)
---|---|---|---|
Competition| Organise the competiton | competition-committee@studentrobotics.org| Wednesday 7:30pm
Kit| They do the kit | kit-committee@studentrobotics.org | Tuesday 7pm
Marketing| Social Media + crocs | marketing-committee@studentrobotics.org| Thursday 7:30pm

> Infrastructure| Hit servers | infrastructure-committee@studentrobotics.org| Wednesday 7pm (monthly)

Reviewer: ```suggestion
Infrastructure| Hit servers | infrastructure-committee@studentrobotics.org| Monday 7:30pm (monthly)
```

> Volunteer Onboarding| Welcome wagon | onboarding@studentrobotics.org | TBC

Reviewer: ```suggestion
Volunteer Onboarding| Welcome wagon | onboarding@studentrobotics.org | Tuesday 8pm (fortnightly)
```

> Fundraising| Moollah | trustees@studentrobotics.org| Mondays 8:30pm

Reviewer: ```suggestion
Fundraising| Moollah | trustees@studentrobotics.org| Monday 8:30pm

```

If you want to help out throughout the year, email the relevant committee or ask in Slack. No prior expereience is necessary, just a can do attitude!

## Stay up to date

If you want to keep up to date with the latest happenings in Student Robotics then you can join our Slack. Most planning and ad-hoc communication occurs there and you'll be able to get stuck in if you want to!

> You could also subscribe to SR(A)WN, our (almost) Weekly Newsletter. This contains headline updates from each Team and can be subscribed to via RSS or Google Groups.

Reviewer: ```suggestion
You could also subscribe to [SR(A)WN](https://studentrobotics.org/srawn/), our (almost) Weekly Newsletter. This contains headline updates from each Team and can be subscribed to via [RSS](https://studentrobotics.org/srawn/rss.xml) or [Google Groups](https://groups.google.com/g/srawn).

```

## Key Dates / Programme Structure

Our experiment with adding a virtual element to the physical competition last year was a success, so we're building on what we've learnt and will continue to have both a physical and virtual competition.

We've pencilled in some key dates for the year with some of the key ones listed below. Be sure to check out the SR shared calendar for all the planned dates including Tech Days and prep events.

- 2023-10-21 Kickstart
- 2024-02-10 Virtual competition (provisional)
- 2024-03-23 – 2024-03-24 Competition (provisional)

## Get involved

There's loads of ways to help with SR2024, taking as much or as little time as you can give.

- The physical competition needs a small army of volunteers to run in numerous techincal and non technical roles. We'll open sign ups for volunteering at the event as soon as we can confirm the venue ???
- We're on the look out for mentors, who would regularly attend a teams' meetings (either virtually on in person) and assist them
- We'll be running Tech Days and in person events which will need people to help run

> - We'll soon be packing the kits for shipping which will require a lot of hands (no technical knowledge required!)

Reviewer 2: ```suggestion
```
Delete this because it's now in the past

Keep an eye on your inboxes for information about these opportunities as details are announced!

### Teams

Running Student Robotics is no small feat so to help spread the load we've organised into teams, each with their own area of interest.

Committee| Description | Email | Current Meeting Time (UK time)
---|---|---|---|
Competition| Organise the competiton | competition-committee@studentrobotics.org| Wednesday 7:30pm
Kit| They do the kit | kit-committee@studentrobotics.org | Tuesday 7pm
Marketing| Social Media + crocs | marketing-committee@studentrobotics.org| Thursday 7:30pm
Infrastructure| Hit servers | infrastructure-committee@studentrobotics.org| Wednesday 7pm (monthly)
Volunteer Onboarding| Welcome wagon | onboarding@studentrobotics.org | TBC
Fundraising| Moollah | trustees@studentrobotics.org| Mondays 8:30pm

If you want to help out throughout the year, email the relevant committee or ask in Slack. No prior expereience is necessary, just a can do attitude!

## Stay up to date

> If you want to keep up to date with the latest happenings in Student Robotics then you can join our Slack. Most planning and ad-hoc communication occurs there and you'll be able to get stuck in if you want to!

Reviewer 2: ```suggestion
If you want to keep up to date with the latest happenings in Student Robotics then you can [join our Slack](https://forms.gle/DsM45qByLA3heZELA). Most planning and ad-hoc communication occurs there and you'll be able to get stuck in if you want to!

```
